### ISSUE TITLE
OCR does not work in background

### ISSUE THREAD

**Issue by Moini** [openpaperwork/paperwork#723]
When I select a document, then select to redo OCR (actually, OCR seems to run on the documents for the first time then, see https://github.com/openpaperwork/paperwork/issues/722), then switch to another document, OCR of that document stops, and is only resumed when I reselect that document (at least, it looks like that to me).

This is with paperwork 1.2.2 (flatpak) on Linux Mint 18.3.

**Comment by jflesch** [comment]
Nah, don't worry, it keeps working in the background. Actually, there is even no way at all for Paperwork to suspend/stop the OCR (no API in Pyocr for that).
You can make sure of that by running `top` in another terminal (or `gnome-system-monitor` for instance). You should see a process `tesseract` using 100% of one of your CPU.

**Comment by Moini** [comment]
I get a different thing in top:

Click on document, select redo ocr, confirm. tesseract starts to run.
Click on second document. tesseract still runs for a couple of seconds. Then stops.
Re-select first document, tesseract starts running again. Animation with pages changing size and paper plane again. Only then tesseract finishes.

**Comment by jflesch** [comment]
Does that happen right after you tried importing documents, or did you restart Paperwork in the meantime ?

**Comment by Moini** [comment]
I had to restart it, because it wouldn't show any documents, probably due to that 'too many open files' issue.
